### PR TITLE
fix: Duration Picker added for Session Types WS-5

### LINKED ISSUE #5699: Wizard Step 5: Session Types appear like time instead of duration
In Wizard Step 5 the session tyes appear like time instead of duration. Please change this as follows:
* Change "Session Types" to -> "Session Types (duration, max. duration 24 hours)"
* The dropdown menu shows duration like times, e.g.. 23:00, it there a clearer way to show that this is about duration and not time of the day?

Screenshot from 2020-11-20 15-39-23

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #5699

#### Short description of what this resolves:
Session Types appear like time instead of duration

#### Changes proposed in this pull request:
- Duration Picker is added to show time like 3.45 instead of 03.45

Screenshot - 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by sachinchauhan2889 — COMMENTED** [review]
sessions and speakers tab in event creation wizard is not even opening.
console is showing this.

sa1

**Comment by MeghalBisht** [comment]
> sessions and speakers tab in event creation wizard is not even opening.
> console is showing this.
> 
> sa1

Are you sure? I am not getting such error

> sessions and speakers tab in event creation wizard is not even opening.
> console is showing this.
> 
> sa1

Are you sure? I am not getting any such error

**Comment by MeghalBisht** [comment]
> sessions and speakers tab in event creation wizard is not even opening.
> console is showing this.

Are you sure? I am not getting any such error.

File: app/components/widgets/forms/duration-picker.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  actions: {
+    onChange() {
+      if (this.onChange) {
+        this.sendAction('onChange', this.value);
+      }
+    }
+  }
+
+});
```
Comment (maze-runnar) [inline app/components/widgets/forms/duration-picker.js:42]: so this file is totally similar to time-picker, only difference is` format        : FORM_DURATION_FORMAT,` if I am not wrong.

So try to pass format as prop in the timePicker for using it as durationPicker.

Comment (MeghalBisht) [reply app/components/widgets/forms/duration-picker.js:42]: Sounds good. Will do that, also are you getting any console errors due to this PR?

Comment (maze-runnar) [reply app/components/widgets/forms/duration-picker.js:42]: I didn't check that, will see.

File: app/templates/components/widgets/forms/duration-picker.hbs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{{#if this.icon}}
+  <i class="time icon"></i>
+{{/if}}
+<div class="ui input">
+  <Input @type="text" @value={{value}} placeholder={{this.placeholder}} @name={{name}} @focus-out={{action "onChange"}} />
+</div>
```
Comment (maze-runnar) [inline app/templates/components/widgets/forms/duration-picker.hbs:6]: delete this file, not needed I guess

**Comment by maze-runnar** [comment]
@MeghalBisht this time format is not allowed in server.
Screenshot from 2020-12-04 14-33-45

**Comment by MeghalBisht** [comment]
> @MeghalBisht this time format is not allowed in server.

Okay, I think without server code being changed this issue cannot be solved?

**Comment by iamareebjamal** [comment]
It can. And server should not be modified to accomodate non-standard display formats from frontend

**Comment by MeghalBisht** [comment]
> It can. And server should not be modified to accomodate non-standard display formats from frontend

Okay, please tell how as these changes are not correct I guess?

**Comment by iamareebjamal** [comment]
There is no requirement for the display value to be sent to the server. Make the changes such that server receives old format and new format is displayed to the user

**Review by sachinchauhan2889 — COMMENTED** [review]
@MeghalBisht you haven't followed up on this PR from last 25 days. PR is pending for a long time. Can you complete this PR by this week, we want to merge it by this week.

**Pull request closed without merging**